### ISSUE TITLE
Servers list

### ISSUE THREAD

**Issue by florian-dubersten** [fabiang/true-combat#3]
Great job!

2 things (windows):
- If you use the filters for the servers list, then servers will not appear anymore, you have to create a new profile to restore this list.
- Is it possible to have a shortcut with the TCE icon that you can place anywhere?  i didn't find a solution.

Also if you need help with 2D design on this project, do not hesitate to contact me.

Hope you understand my poor english.

Thanks.

**Comment by fabiang** [comment]
Hello,

thanks that you're interested in playing good ol' True Combat:Elite.

1. I don't know how to fix that issue with the server list. If you find a solution please provide feedback
2. Did you run the install.exe? It should have created a shortcut for you: `TCE.lnk`

And I don't need any 2D work, since this project was finished over ten years ago and I'm no the original developer. You can find them here: http://www.truecombatelite.com/

**Comment by florian-dubersten** [comment]
Yes, i run the install.exe, it creates a TCE shortcut (without the ".lnk" extension) with the ET icon (that is why i proposed my help with 2D design) but when i do a drag and drop on my desktop, it only opens ET console, saying "couldn't load defaut.cfg".

I know you are not Coroner, but i found your project very usefull (since TCE website  does not have anymore links to download TCE files). 

Thx again.

**Comment by fabiang** [comment]
Ok maybe the shortcut should define the path to `default.cfg` and it could be fixed. Guess we have a bug here.
